<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Table_Companies">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>300</width>
      <height>420</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>299</number>
    </attribute>
    <column>
     <property name="text">
      <string>Companies</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="Table_JobOpeners">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>400</width>
      <height>420</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>400</number>
    </attribute>
    <column>
     <property name="text">
      <string>Job Openings</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="Table_Condidates">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>560</width>
      <height>420</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>575</number>
    </attribute>
    <column>
     <property name="text">
      <string>Candidates</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>260</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add job</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>460</y>
      <width>510</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add candidate</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_compNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CompanyName:</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_compNameChanged">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Apple</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_postLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Post:</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_postChanged">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Programmer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_postLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Salary:</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_salaryEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_postLabel_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Is Have High Education:</string>
    </property>
   </widget>
   <widget class="QLabel" name="JOBSHOW_EducEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>true</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_postLabel_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Post:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_postChanged_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Programmer</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_salaryEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_compNameLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FIO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_postLabel_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Minimum Salary:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_postLabel_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Is Have High Education:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_EducEdit_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>true</string>
    </property>
   </widget>
   <widget class="QLabel" name="CONDIDATE_compNameChanged_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Apple</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
